src/cdk/testing/protractor/protractor-element.ts

```diff
@@ -73,9 +73,10 @@ export class ProtractorElement implements TestElement {
     return this.element.clear();
   }
 
-  async click(relativeX = 0, relativeY = 0): Promise<void> {
+  async click(...args: number[]): Promise<void> {
+    const offsetArgs = args.length ? [{x: args[0], y: args[1]}] : [];
```

Comment:
Optional nit: Might be good to have a super-short comment saying that "omitting the last argument of `mouseMove` results in the center of the element being used.